mobile-academy/src/main/java/org/motechproject/nms/mobileacademy/domain/CompletionRecord.java

```diff
@@ -25,10 +25,13 @@
     private int score;
 
     @Field
+    private int completionCount;
+
+    @Field
     private boolean sentNotification;
 
     @Field
-    private int completionCount;
+    private String lastDeliveryStatus;
```

Comment:
No more. All the remaining items (2) have github issues tracking them. #277
#278 and the motech ticket for hibernate validation

On Tue, May 26, 2015 at 9:47 AM, Lauren Lavoie <notifications@github.com>
wrote:

> In
> mobile-academy/src/main/java/org/motechproject/nms/mobileacademy/domain/CompletionRecord.java
> <https://github.com/motech-implementations/mim/pull/269#discussion_r31054232>
> :
>
> >      private boolean sentNotification;
> >
> >      @Field
> > -    private int completionCount;
> > +    private String lastDeliveryStatus;
>
> Ok. Do you have any more to-dos left on this PR or shall I merge?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/motech-implementations/mim/pull/269/files#r31054232>.
>